web-api/run-umzug.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -e
+ENV=$1
+node ./web-api/umzug.js $ENV
```

Comment:
```suggestion
node ./web-api/umzug.js "$ENV"
```
I'm pretty sure this should be quoted.